### ISSUE TITLE
read.odf() complains about CRAT unit "(none)"

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1971]
The warning
```
  "conductivity" (code name "CRAT_01") is a conductivity ratio, which has no units, but the file lists "(none)" as a unit. Consult ?read.odf to see how to rectify this error.
```
is generated from an ODF file witih a data fragment as below.  I will change this to accept `"(none)"` as being appropriate.
```
PARAMETER_HEADER,
  TYPE='SING',
  NAME='Conductivity Ratio',
  UNITS='(none)',
  CODE=CRAT_01,
  NULL_VALUE=-99.00,
  PRINT_FIELD_WIDTH=10,
  PRINT_DECIMAL_PLACES=6,
  ANGLE_OF_SECTION=0.000000,
  MAGNETIC_VARIATION=0.000000,
  DEPTH=1158.000000 ,
  MINIMUM_VALUE=0.759329,
  MAXIMUM_VALUE=0.765117,
  NUMBER_VALID=105973,
  NUMBER_NULL=0 
```

**Comment by dankelley** [comment]
Fixed in "develop" commit f34000393c93dca076d4cc50ec59907a7189b8e7.  The recognized codes are now as defined in the few lines starting at https://github.com/dankelley/oce/blob/f34000393c93dca076d4cc50ec59907a7189b8e7/R/odf.R#L1541

**Comment by dankelley** [comment]
This passes local tests, and also it works on some MCM files generated from BIO measurements on Sackville Spur.
